includes/pet-details-form.php

```diff
@@ -169,3 +170,5 @@
         </svg>
     </span>
 </p>
+<button>Submit</button>
+</form>
```

Comment:
Have removed this on latest commit - good idea. Also means if we pass a variable for $status like 'lost' and 'found' in to the page then we could grab that in the function and use the same function for both submission pages